lms/djangoapps/discussion_api/api.py

```diff
@@ -21,8 +21,7 @@
     can_delete,
     get_editable_fields,
     get_initializable_comment_fields,
-    get_initializable_thread_fields,
-)
+    get_initializable_thread_fields)
```

Comment:
@wajeeha-khalid Python allows trailing commas like this.
We tend to leave trailing commas so any future additions to the list will not include this last line in the code-diff.  This makes it easier to code review.